src/action/Button/Button.tsx

```diff
@@ -139,19 +158,71 @@ const buttonKind = (kind: Kind = "normal", variant: Variant = "primary") => {
       return css`
         background-color: transparent;
         color: ${color};
-        border: 0;
+        border: 2px solid transparent;
 
         &:hover:enabled {
           opacity: 0.8;
         }
       `;
+    case "text":
+      return css`
+        background-color: transparent;
+        color: ${color};
+        border: 2px solid transparent;
+
+        &:hover {
+          background-color: ${colorLight};
```

Comment:
why not using **theme.color[variantLight]** directly, if it's only used here ?

Reply:
Modified